src/SlackGhost.ts

```diff
@@ -269,7 +269,7 @@ export class SlackGhost {
             return;
         }
 
-        const match = hash || avatarUrl.match(/\/([^\/]+)$/);
```

Comment:
We should be mindful here that this is a breaking change too?

Reply:
`/` inside of `[]` is already a literal `/`